client/my-sites/plans-features-main/index.tsx

```diff
@@ -92,12 +107,36 @@ export interface PlansFeaturesMainProps {
 	withDiscount?: string;
 	discountEndDate?: Date;
 	hidePlansFeatureComparison?: boolean;
-	hideFreePlan?: boolean; // to be deprecated
-	hidePersonalPlan?: boolean; // to be deprecated
-	hidePremiumPlan?: boolean; // to be deprecated
-	hideBusinessPlan?: boolean; // to be deprecated
-	hideEcommercePlan?: boolean; // to be deprecated
-	hideEnterprisePlan?: boolean; // to be deprecated
+
+	/**
+	 * @deprecated use intent mechanism instead
```

Comment:
love it. hopefully we'll manage to get rid of these and centralize either behind intents or through the Oboarding data store (that includes similar setters/getters). There are use cases that may need them, so it needs some thought. Not sure if anything that can't be centralized behind plans-intent mechanism